css/cssom-view/MediaQueryList-addListener-handleEvent.html

```diff
@@ -72,21 +72,26 @@
 }, "doesn't look up handleEvent method on callable event listeners");
 
 const uncaught_error_test = async (t, mql, getHandleEvent) => {
+    const eventWatcher = new EventWatcher(t, window, "error", waitForChangesReported);
+    const errorPromise = eventWatcher.wait_for("error");
+
     let calls = 0;
-    mql.addListener({
+    const listener = {
         get handleEvent() {
             calls++;
             return getHandleEvent();
         },
-    });
+    };
 
-    const eventWatcher = new EventWatcher(t, window, "error", waitForChangesReported);
-    const errorPromise = eventWatcher.wait_for("error");
-    triggerMQLEvent(mql);
+    try {
+        mql.addListener(listener);
```

Comment:
This commit is follow-up of https://github.com/web-platform-tests/wpt/pull/21186#discussion_r366813367 (sorry).

`uncaught_error` tests are false-positive in WebKit because `addListener` throws on non-callable `listener`. I am moving `assert_equals(calls` to `finally` clause so it would execute.

//cc @zcorpan

Reply:
This is a legitimate bug in webkit, right? Why not let it fail?

Reply:
I might have used "false-positive" wrong, sorry.

Currently, WebKit passes `uncaught_error tests` tests even though it doesn't support `addListener` with an object. This change makes WebKit fail these tests.

WebKit bug: [MediaQueryList should extend EventTarget](https://bugs.webkit.org/show_bug.cgi?id=203288).

Reply:
Either that or I misread it. Thanks for the clarification, this seems fine.